templates/partials/style.css

```diff
@@ -6,7 +6,7 @@
 
 .emailBody a{
   text-decoration: none !important;
-  color: #1C58D9 !important;
```

Comment:
This prevented overrides of a lot of things. Dropped !important

Reply:
Will this have any side effect to other emails?

Reply:
Not from what I could tell when I tried. But they're emails, it's always possible something messes up